Configuration/PyReleaseValidation/python/relval_gpu.py

```diff
@@ -13,70 +13,65 @@
-#           Patatrack ECAL-only:                                TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
-#           Patatrack HCAL-only:                                TTbar - on GPU (optional), GPU-vs-CPU validation, profiling
-#           Patatrack pixel-only quadruplets, ECAL, HCAL:       TTbar - on GPU (optional), GPU-vs-CPU validation
-#           Patatrack pixel-only triplets, ECAL, HCAL:          TTbar - on GPU (optional), GPU-vs-CPU validation
-#           full reco with Patatrack pixel-only quadruplets:    TTbar - on GPU (optional), GPU-vs-CPU validation
-#           full reco with Patatrack pixel-only triplets:       TTbar - on GPU (optional), GPU-vs-CPU validation
-#           Patatrack pixel-only quadruplets:                   Single Nu E10 on GPU (optional)
+# with PU
 #           Alpaka pixel-only quadruplets:                      TTbar with PU: any backend, any backend vs cpu validation, profiling
 #           Alpaka ECAL-only:                                   TTbar with PU: any backend
 #           Alpaka HCAL-only:                                   TTbar with PU: any backend, any backend vs cpu validation, profiling
 #           Alpaka with full reco and pixel-only quadruplets:   TTbar with PU: any backend
-#           Alpaka pixel-only quadruplets:                      ZMM with PU: any backend, any backend vs cpu validation, profiling
+#           Alpaka pixel-only:                                  ZMM with PU: any backend, any backend vs cpu validation, profiling, triplets
 #           Alpaka pixel-only quadruplets:                      Single Nu E10 with PU: any backend
-# mc 2026   Patatrack pixel-only quadruplets:                   Single Nu E10: on GPU (optional)
+# mc 2026   
+# no PU
+#           Alpaka pixel-only:                                  TTbar: quadruplets any backend, any backend vs cpu validation, profiling, triplets                   
+#           Alpaka pixel-only:                                  Single Nu E10: on GPU (optional)
+# with PU
+#           Alpaka pixel-only:                                  TTbar: quadruplets any backend, any backend vs cpu validation, profiling, triplets                   
+#           Alpaka pixel-only:                                  Single Nu E10: on GPU (optional)
+
 numWFIB = [
-           # 2023, Alpaka-based
-           12434.402, 12434.403, 12434.404,
-          #12434.406, 12434.407, 12434.408,
-           12434.412,#12434.413, 12434.414,
-           12434.422, 12434.423, 12434.424,
-          #12434.482, 12434.483, 12434.484
-          #12434.486, 12434.487, 12434.488
-           12434.492,#12434.493
-           12450.402, 12450.403, 12450.404,
+           # 2023, Alpaka-based noPU
+           12434.402,12434.403,12434.412,12434.422,12434.423
+
+           # 2024, Alpaka-based noPU
+           12834.402, 12834.403, 12834.404, 12834.406,
+          #12834.406, 12834.407, 12834.408,
```

Comment:
```suggestion
           12834.402, 12834.403, 12834.404,
           12834.406,#12834.407, 12834.408,
```

Reply:
Actually, since you have introduced the customisation for the triplets, can you define also `.407` and `.408` based on `.403` and `.404` ?

Reply:
The same comment applies to the other instances of `.406` below.